Arm/Cosim.lean

```diff
@@ -109,8 +115,7 @@ def arm_cosim_test (input : regState) (guid : Nat) : IO String := do
   let gprs'   := bitvec_to_hex_list input.gpr
   let flags'  := bitvec_to_hex input.nzcv
   let sfps'   := bitvec_to_hex_list input.sfp
-  let instructionStr := ([inst'] ++ gprs' ++ [flags'] ++ sfps')
-  let args    := [toString guid] ++ instructionStr |>.toArray
+  let args    := [uniqueBaseName] ++ [inst'] ++ gprs' ++ [flags'] ++ sfps' |>.toArray
```

Comment:
```suggestion
  let args    := [uniqueBaseName, inst'] ++ gprs' ++ [flags'] ++ sfps' |>.toArray
```

Reply:
@alexkeizer I disagree, the current ordering is "semantic" since the base name semantically belongs to a different "category"